<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<link href="default.css" type="text/css" rel="stylesheet">
<script language="javascript" type="text/javascript">
function update_modtime() {
  var x = new Date(document.lastModified);
  document.getElementById("last_mod").innerHTML="Last modified: "+x;
}
</script>
<title>Chongzhi Zang</title>
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
</head>
<body bgcolor="white" onload="update_modtime();">
<div style="width: 800px; margin: auto;">
<img align="right" src="cz_large.jpg" style="width:200px">
<h2><font color="#002F6C">Chongzhi Zang, Ph.D.</font></h2>
<p>Assistant Professor of Public Health Sciences<br>
Assistant Professor of Biomedical Engineering<br>
Assistant Professor of Biochemistry and Molecular Genetics<br></p><p>
<a href="http://cphg.virginia.edu/" target="_blank" style="text-decoration:none;">Center for Public Health Genomics</a><br>
<a href="http://www.virginia.edu/" target="_blank" style="text-decoration:none;">University of Virginia</a> <a href="https://med.virginia.edu/" target="_blank" style="text-decoration:none;">School of Medicine</a><br>
P. O. Box 800717, 
Charlottesville, VA 22908<br>
</p>
<p>
Office: <a href="http://www.virginia.edu/webmap/popPages/130-westcomplex.html" target="_blank" style="text-decoration:none;">West Complex (MSB)</a> 6131C<br>
Phone: 434-243-5397<br>
Email: <a href="mailto:zang@virginia.edu">zang@virginia.edu</a>
</p>
<p>
CV: <a href="CV_chongzhi_zang.pdf" target="_blank">download</a><br>
Lab website: <a href="http://faculty.virginia.edu/zanglab" target="_blank">faculty.virginia.edu/zanglab</a>
</p>
<hr>
<h3><font color="#002F6C">Education and Training</font></h3>
<p>
B.S., Physics, <a href="http://www.pku.edu.cn/" target="_blank" style="text-decoration:none;">Peking University</a>, 2005<br></p><p>
Ph.D., Physics, <a href="http://www.gwu.edu/" target="_blank" style="text-decoration:none;">The George Washington University</a>, 2010<br></p><p>
Postdoc, Biostatistics and Computational Biology, <a href="http://www.dana-farber.org/" target="_blank" style="text-decoration:none;">Dana-Farber Cancer Institute</a>, <a href="http://www.harvard.edu/" target="_blank" style="text-decoration:none;">Harvard University</a>, 2010&ndash;2016<br>
</p><hr>
<h3><font color="#002F6C">Research Interests</font></h3>
<p>Bioinformatics methodology development; Epigenetics and chromatin biology; Transcriptional regulation; Cancer genomics and epigenomics; Statistical methods for biomedical data integration; Theoretical and computational biophysics
</p><hr>
<h3><font color="#002F6C">Research Description</font></h3>
<p>The research in my lab focuses on developing computational methodologies and integrative genomics approaches to study epigenetics and transcriptional regulation of gene expression in a variety of mammalian cell systems and human diseases such as cancer. 
</p><p>
How gene expression is regulated in chromatin is a fundamental question in molecular biology. High-throughput technologies such as next-generation sequencing (NGS) have become powerful tools for studying gene regulation at the genomic scale. We conduct computational research that leverages these genomics technologies. Some research directions include: 
</p><p>
1. <i>Next-generation sequencing bioinformatics</i> 
</p><p>
We are interested in developing statistical methods and novel algorithms for analyzing massive data from next-generation sequencing (NGS) coupled with various assays for studying genomic chromatin profiles, such as ChIP-seq for transcription factor and histone mark profiling, ATAC-seq and DNase-seq for chromatin accessibility profiling, etc. As a pioneer in ChIP-seq bioinformatics, we developed SICER (<a href="https://academic.oup.com/bioinformatics/article/25/15/1952/212783" target="_blank"><i>Bioinformatics</i> 2009</a>), one of the most widely used methods for ChIP-seq data analysis with exceptional performance for board histone modification marks. We are developing novel statistical models and computational methods for analyzing DNase/ATAC-seq data and for studying chromatin dynamics. 
</p><p>
2. <i>Chromatin, epigenetics, and transcriptional regulation</i> 
</p><p>
Our ultimate goal is to understand the fundamental mechanisms in transcriptional regulation and the functions of chromatin. We characterized dozens of histone modifications and histone modifying enzymes at the genomic scale (<a href="http://www.nature.com/ng/journal/v40/n7/full/ng.154.html" target="_blank"><i>Nat Genet</i> 2008</a>, <a href="http://www.nature.com/ng/journal/v41/n8/full/ng.409.html" target="_blank"><i>Nat Genet</i> 2009</a>, <a href="http://www.cell.com/abstract/S0092-8674(09)00841-1" target="_blank"><i>Cell</i> 2009</a>, <a href="http://www.cell.com/cell-stem-cell/abstract/S1934-5909(08)00582-1" target="_blank"><i>Cell Stem Cell</i> 2009</a>). Leveraging the large amount of publicly available ChIP-seq data, we developed MARGE (<a href="http://genome.cshlp.org/content/26/10/1417" target="_blank"><i>Genome Res</i> 2016</a>), a computational method to predict cis-regulatory profiles from differential expression gene sets using integrative learning approaches. We are specifically interested in studying functional enhancer regulation of gene expression in cancers.
</p><p>
3. <i>Genomic data integration for chromatin dynamics and regulatory networks</i> 
</p><p>
High-dimensional genomic data analysis is challenging because of noises and biases in high-throughput experiments. We developed MANCIE (<a href="http://www.nature.com/articles/ncomms11305" target="_blank"><i>Nat Commun</i> 2016</a>), a method for bias correction and data integration of cross-platform genomic profiles on the same samples, using a Bayesian-supported principal component analysis (PCA)-based approach. We are interested in using statistical modeling and machine-learning approaches to integrate public genomic data for characterizing physical properties of mammalian epigenomes and dynamic interactions between chromatin and DNA in human cell systems.
</p><hr>
<h3><font color="#002F6C">Selected Publications</font></h3>
<p>
Selected from 30+ journal articles. A complete publication list can be found at my <a href="https://scholar.google.com/citations?user=GV7vSDMAAAAJ" target="_blank">Google Scholar profile</a>.<br>
* indicates authors with equal contributions.</p>
<ol style="padding-left: 24px;">
<li><p><a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty194/4956015" target="_blank">BART: a transcription factor prediction tool with query gene sets or epigenomic profiles</a><br> 
Zhenjia Wang, Mete Civelek, Clint L. Miller, Nathan C. Sheffield, Michael J. Guertin, <u>Chongzhi Zang</u><br>
<i><b>Bioinformatics</b></i> 34, 2867&ndash;2869 (2018)
</p></li>
<li><p><a href="https://doi.org/10.1073/pnas.1722617115" target="_blank">Estrogen-regulated feedback loop limits the efficacy of estrogen receptor–targeted breast cancer therapy</a><br> 
Tengfei Xiao, Wei Li, Xiaoqing Wang, Han Xu, Jixin Yang, Qiu Wu, Ying Huang, Joseph Geradts, Peng Jiang, Teng Fei, David Chi, <u>Chongzhi Zang</u>, Qi Liao, Jonathan Rennhack, Eran Andrechek, Nanlin Li, Simone Detre, Mitchell Dowsett, Rinath M. Jeselsohn, X. Shirley Liu, Myles Brown<br>
<i><b>Proceedings of the National Academy of Sciences USA</b></i>, doi:10.1073/pnas.1722617115 (2018)
</p></li>
<li><p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/mc.22802" target="_blank">Nickel exposure induces persistent mesenchymal phenotype in human lung epithelial cells through epigenetic activation of ZEB1</a><br> 
Cynthia C Jose, Lakshmanan Jagannathan, Vinay Singh Tanwar, Xiaoru Zhang, <u>Chongzhi Zang</u>, Suresh Cuddapah<br>
<i><b>Molecular Carcinogenesis</b></i> 57, 794&ndash;806 (2018)
</p></li>
<li><p><a href="http://cancerres.aacrjournals.org/content/77/21/e19" target="_blank">Cistrome Cancer: a web resource for integrative gene regulation modeling in cancer</a><br> 
Shenglin Mei, Clifford A. Meyer, Rongbin Zheng, Qian Qin, Qiu Wu, Peng Jiang, Bo Li, Xiaohui Shi, Binbin Wang, Jingyu Fan, Celina Shih, Myles Brown, <u>Chongzhi Zang</u><sup>§</sup>(co-corresponding author), X. Shirley Liu<sup>§</sup><br>
<i><b>Cancer Research</b></i> 77, e19&ndash;e22 (2017)
</p></li>
<li><p><a href="https://academic.oup.com/nar/article/doi/10.1093/nar/gkx1053/4602870" target="_blank">Universal correction of enzymatic sequence bias reveals molecular signatures of protein/DNA interactions</a><br> 
André L. Martins, Ninad M. Walavalkar, Warren D. Anderson, <u>Chongzhi Zang</u>, Michael J. Guertin<br>
<i><b>Nucleic Acids Research</b></i> 46 (2), e9 (2018)
</p></li>
<li><p><a href="http://stke.sciencemag.org/content/10/477/eaag1598" target="_blank">Genome-wide identification and characterization of Notch transcription complex-binding sequence-paired sites in leukemia cells</a><br> 
Eric Severson*, Kelly L. Arnett*, Hongfang Wang*, <u>Chongzhi Zang</u>*, Len Taing, Hudan Liu, Warren S. Pear, X. Shirley Liu, Stephen C. Blacklow, Jon C. Aster<br>
<i><b>Science Signaling</b></i> 10, 477, eaag1598 (2017)
</p></li>
<li><p>
<a href="http://www.nature.com/articles/ncomms11305" target="_blank">High-dimensional genomic data bias correction and data integration using MANCIE</a><br> 
<u>Chongzhi Zang</u>*, Tao Wang*, Ke Deng, Bo Li, Sheng’en Hu, Qian Qin, Tengfei Xiao, Shihua Zhang, Clifford A. Meyer, Housheng Hansen He, Myles Brown, Jun S. Liu, Yang Xie, X. Shirley Liu<br>
<i><b>Nature Communications</b></i> 7, 11305 (2016)
</p></li><li><p>
<a href="http://genome.cshlp.org/content/26/10/1417" target="_blank">Modeling cis-regulation with a compendium of genome-wide histone H3K27ac profiles</a><br>
Su Wang*, <u>Chongzhi Zang</u>*, Tengfei Xiao, Jingyu Fan, Shenglin Mei, Qian Qin, Qiu Wu, Xujuan Li, Kexin Xu, Housheng Hansen He, Myles Brown, Clifford A. Meyer, X. Shirley Liu<br>
<i><b>Genome Research</b></i> 26, 1417&ndash;1429 (2016)
</p></li><li><p>
<a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkw983" target="_blank">Cistrome Data Browser: an integrated data portal for ChIP-Seq and chromatin accessibility data in human and mouse</a><br>
Shenglin Mei, Qian Qin, Qiu Wu, Hanfei Sun, Rongbin Zheng, <u>Chongzhi Zang</u>, Muyuan Zhu, Jiaxin Wu, Xiaohui Shi, Len Taing, Tao Liu, Myles Brown, Clifford A. Meyer, X. Shirley Liu<br>
<i><b>Nucleic Acids Research</b></i> 45 (D1), D658&ndash;D662 (2016) 
</p></li><li><p>
<a href="http://www.nature.com/articles/srep30255" target="_blank">NF-E2, FLI1 and RUNX1 collaborate at areas of dynamic chromatin to activate transcription in mature mouse megakaryocytes</a><br>
<u>Chongzhi Zang</u>*, Annouck Luyten*, Christina Chen, X. Shirley Liu, Ramesh A. Shivdasani<br>
<i><b>Scientific Reports</b></i> 6, 30255 (2016)
</p></li><li><p>
<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1274-4" target="_blank">ChiLin: a comprehensive ChIP-seq and DNase-seq quality control and analysis pipeline</a><br>
Qian Qin, Shenglin Mei, Qiu Wu, Hanfei Sun, Lewyn Li, Len Taing, Sujun Chen, Fugen Li, Tao Liu, <u>Chongzhi Zang</u>, Han Xu, Yiwen Chen, Clifford A. Meyer, Yong Zhang, Myles Brown, Henry W. Long, X. Shirley Liu<br>
<i><b>BMC Bioinformatics</b></i> 17, 404 (2016)
</p></li><li><p>
<a href="http://mcr.aacrjournals.org/content/14/2/163" target="_blank">Integrative analysis reveals the transcriptional collaboration between EZH2 and E2F1 in the regulation of cancer-related gene expression</a><br>
Han Xu*, Kexin Xu*, Housheng H. He, <u>Chongzhi Zang</u>, Chen-Hao Chen, Yiwen Chen, Qian Qin, Su Wang, Chenfei Wang, Sheng’en Hu, Fugen Li, Henry Long, Myles Brown, X. Shirley Liu<br>
<i><b>Molecular Cancer Research</b></i> 14, 163&ndash;172 (2016)
</p></li><li><p>
<a href="http://www.nature.com/ng/journal/v47/n11/full/ng.3404.html" target="_blank">Partitioning heritability by functional annotation using genome-wide association summary statistics</a><br>
Hilary K. Finucane, Brendan Bulik-Sullivan, Alexander Gusev, Gosia Trynka, Yakir Reshef, Po-Ru Loh, Verneri Anttila, Han Xu, <u>Chongzhi Zang</u>, Kyle Farh, Stephan Ripke, Felix R. Day, ReproGen Consortium, Schizophrenia Working Group of the Psychiatric Genomics Consortium, The RACI Consortium, Shaun Purcell, Eli Stahl, Sara Lindstrom, John R. B. Perry, Yukinori Okada, Soumya Raychaudhuri, Mark J. Daly, Nick Patterson, Benjamin M. Neale, Alkes L. Price<br>
<i><b>Nature Genetics</b></i> 47, 1228&ndash;1235 (2015)
</p></li><li><p>
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0808-9" target="_blank">Network analysis of gene essentiality in functional genomics experiments</a><br>
Peng Jiang, Hongfang Wang, Wei Li, <u>Chongzhi Zang</u>, Bo Li, Yinling J. Wong, Clifford Meyer, Jun S. Liu, Jon C. Aster, X. Shirley Liu<br>
<i><b>Genome Biology</b></i> 16, 239 (2015)
</p></li><li><p>
<a href="http://onlinelibrary.wiley.com/doi/10.1002/jcp.24872/abstract" target="_blank">The role of Notch receptors in transcriptional regulation</a><br>
Hongfang Wang, <u>Chongzhi Zang</u>, X. Shirley Liu, Jon C. Aster<br>
<i><b>Journal of Cellular Physiology</b></i> 230, 982&ndash;988 (2015)
</p></li><li><p>
<a href="http://www.pnas.org/content/111/2/705.long" target="_blank">NOTCH1-RBPJ complexes drive target gene expression through dynamic interactions with superenhancers</a><br>
Hongfang Wang*, <u>Chongzhi Zang</u>*, Len Taing, Kelly Arnett, Yinling Joey Wong, Warren S. Pear, Stephen C. Blacklow, X. Shirley Liu, Jon C. Aster<br>
<i><b>Proceedings of the National Academy of Sciences USA</b></i> 111, 715&ndash;710 (2014)
</p></li><li><p>
<a href="http://genesdev.cshlp.org/content/28/16/1827" target="_blank">Active enhancers are delineated de novo during hematopoiesis with limited lineage fidelity among specified primary blood cells</a><br>
Annouck Luyten*, <u>Chongzhi Zang</u>*, X. Shirley Liu, Ramesh A. Shivdasani<br>
<i><b>Genes and Development</b></i> 28, 1827&ndash;1839 (2014)
</p></li><li><p>
<a href="http://www.nature.com/nmeth/journal/v11/n1/full/nmeth.2762.html" target="_blank">Refined DNase-seq protocol and data analysis reveals intrinsic bias in transcription factor footprint identification</a><br>
Housheng Hansen He, Clifford A. Meyer, Sheng’en Shawn Hu, Mei-Wei Chen, <u>Chongzhi Zang</u>, Yin Liu, Prakash K. Rao, Teng Fei, Han Xu, Henry Long, X. Shirley Liu, Myles Brown<br>
<i><b>Nature Methods</b></i> 11, 73&ndash;78 (2014)
</p></li><li><p>
<a href="http://www.cell.com/ajhg/abstract/S0002-9297(14)00426-1" target="_blank">Partitioning heritability of regulatory and cell-type-specific variants across 11 common diseases</a><br>
Alexander Gusev, S. Hong Lee, Gosia Trynka, Hilary Finucane, Bjarni J. Vilhjálmsson, Han Xu, <u>Chongzhi Zang</u>, Stephan Ripke, Brendan Bulik-Sullivan, Eli Stahl, Schizophrenia Working Group of the Psychiatric Genomics Consortium, SWE-SCZ Consortium, Anna K. Kähler, Christina M. Hultman, Shaun M. Purcell, Steven A. McCarroll, Mark Daly, Bogdan Pasaniuc, Patrick F. Sullivan, Benjamin M. Neale, Naomi R. Wray, Soumya Raychaudhuri, Alkes L. Price<br>
<i><b>The American Journal of Human Genetics</b></i> 95, 535&ndash;552 (2014)
</p></li><li><p>
<a href="http://cancerdiscovery.aacrjournals.org/content/4/10/1154.long" target="_blank">Discovery of biomarkers predictive of GSI response in triple negative breast cancer and adenoid cystic carcinoma</a><br>
Alexander Stoeck, Serguei Lejnine, Andrew Truong, Li Pan, Hongfang Wang, <u>Chongzhi Zang</u>, Jing Yuan, Chris Ware, John MacLean, Philip W. Garrett-Engele, Michael Kluk, Jason Laskey, Brian B. Haines, Christopher Moskaluk, Leigh Zawel, Stephen Fawell, D. Gary Gilliland, Theresa Zhang, Brandon Kremer, Birgit Knoechel, Bradley E. Bernstein, Warren S. Pear, X. Shirley Liu, Jon C. Aster, Sriram Sathyanarayanan<br>
<i><b>Cancer Discovery</b></i> 4(10), 1154&ndash;1167 (2014)
</p></li><li><p>
<a href="http://www.pnas.org/content/111/46/E4946.full" target="_blank">Long-range enhancer activity determines Myc sensitivity to Notch inhibitors in T cell leukemia</a><br>
Yumi Yashiro-Ohtani, Hongfang Wang, <u>Chongzhi Zang</u>, Kelly L. Arnett, Will Bailis, Yugong Ho, Birgit Knoechel, Claudia Lanauze, Lumena Louis, Katherine S. Forsyth, Sujun Chen, Yoonjie Chung, Jonathan Schug, Gerd A. Blobel, Stephen A. Liebhaber, Bradley E. Bernstein, Stephen C. Blacklow, X. Shirley Liu, Jon C. Aster, Warren S. Pear<br>
<i><b>Proceedings of the National Academy of Sciences USA</b></i> 111(46), E4946&ndash;E4953 (2014)
</p></li><li><p>
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0419-x" target="_blank">MethylPurify: tumor purity deconvolution and differential methylation detection from single tumor DNA methylomes</a><br>
Xiaoqi Zheng, Qian Zhao, Huajun Wu, Wei Li, Haiyun Wang, Clifford A. Meyer, Qian Alvin Qin, Han Xu, <u>Chongzhi Zang</u>, Peng Jiang, Fuqiang Li, Yong Hou, Jianxing He, Jun Wang, Jun Wang, Peng Zhang, Yong Zhang, X. Shirley Liu<br>
<i><b>Genome Biology</b></i> 15, 419 (2014)
</p></li><li><p>
<a href="http://www.nature.com/nprot/journal/v8/n12/full/nprot.2013.150.html" target="_blank">Targets analysis by integration of transcripome and ChIP-seq data with BETA</a><br>
Su Wang, Hanfei Sun, Jian Ma, <u>Chongzhi Zang</u>, Chenfei Wang, Juan Wang, Qianzi Tang, Clifford A. Meyer, Yong Zhang, X. Shirley Liu<br>
<i><b>Nature Protocols</b></i> 8, 2502&ndash;2515 (2013)
</p></li><li><p>
<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002649" target="_blank">Transcriptional regulation of rod photoreceptor homeostasis revealed by in vivo NRL targetome analysis</a><br>
Hong Hao, Douglas S. Kim, Bernward Klocke, Kory R. Johnson, Kairong Cui, Norimoto Gotoh, <u>Chongzhi Zang</u>, Janina Gregorski, Linn Gieser, Weiqun Peng, Yang Fann, Martin Seifert, Keji Zhao, Anand Swaroop<br>
<i><b>PLoS Genetics</b></i> 8(4), e1002649 (2012)
</p></li><li><p>
<a href="http://science.sciencemag.org/content/329/5994/917.full" target="_blank">PTIP promotes chromatin changes critical for immunoglobulin switch recombination</a><br>
Jeremy A. Daniel, Margarida A. Santos*, Zhibin Wang*, <u>Chongzhi Zang</u>*, Mila Jankovic, Anna Gazumyan, Kristopher R. Schwab, Arito Yamane, Darius Filsuf, Young-Wook Cho, Kai Ge, Weiqun Peng, Michel C. Nussenzweig, Rafael Casellas, Gregory R. Dressler, Keji Zhao, André Nussenzweig<br>
<i><b>Science</b></i> 329, 917&ndash;923 (2010)
</p></li><li><p>
<a href="http://www.cell.com/immunity/abstract/S1074-7613(10)00206-2" target="_blank">Discrete roles of STAT4 and STAT6 transcription factors in tuning epigenetic modifications and transcription during T helper cell differentiation</a><br>
Lai Wei, Golnaz Vahedi, Hong-Wei Sun, Wendy T. Watford, Hiroaki Takatori, Haydee L. Ramos, Hayato Takahashi, Jonathan Liang, Gustavo Gutierrez-Cruz, <u>Chongzhi Zang</u>, Weiqun Peng, John J. O'Shea, Yuka Kanno<br>
<i><b>Immunity</b></i> 32, 840&ndash;851 (2010)
</p></li><li><p>
<a href="https://academic.oup.com/bioinformatics/article/25/15/1952/212783" target="_blank">A clustering approach for identification of enriched domains from histone modification ChIP-Seq data</a><br>
<u>Chongzhi Zang</u>, Dustin E. Schones, Chen Zeng, Kairong Cui, Keji Zhao, Weiqun Peng<br>
<i><b>Bioinformatics</b></i> 25, 1952&ndash;1958 (2009)
</p></li><li><p>
<a href="http://www.cell.com/abstract/S0092-8674(09)00841-1" target="_blank">Genome-wide mapping of HATs and HDACs reveals distinct functions in active and inactive genes</a><br>
Zhibin Wang*, <u>Chongzhi Zang</u>*, Kairong Cui*, Dustin E. Schones, Artem Barski, Weiqun Peng, Keji Zhao<br>
<i><b>Cell</b></i> 138, 1019&ndash;1031 (2009)
</p></li><li><p>
<a href="http://www.nature.com/ng/journal/v41/n8/full/ng.409.html" target="_blank">H3.3/H2A.Z double variant-containing nucleosomes mark ‘nucleosome-free regions’ of active promoters and other regulatory regions</a><br>
Chunyuan Jin*, <u>Chongzhi Zang</u>*, Gang Wei, Kairong Cui, Weiqun Peng, Keji Zhao, Gary Felsenfeld<br>
<i><b>Nature Genetics</b></i> 41, 941&ndash;945 (2009)
</p></li><li><p>
<a href="http://www.cell.com/cell-stem-cell/abstract/S1934-5909(08)00582-1" target="_blank">Chromatin signatures in multipotent hematopoietic stem cells indicate the fate of bivalent genes during differentiation</a><br>
Kairong Cui*, <u>Chongzhi Zang</u>*, Tae-Young Roh, Dustin E. Schones, Richard W. Childs, Weiqun Peng, Keji Zhao<br>
<i><b>Cell Stem Cell</b></i> 4, 80&ndash;93 (2009)
</p></li><li><p>
<a href="http://www.cell.com/immunity/abstract/S1074-7613(09)00235-0" target="_blank">Genome-wide analysis of histone methylations reveals chromatin state-based regulation of gene transcription and function of memory CD8+ T cells</a><br>
Yasuto Araki, Zhibin Wang, <u>Chongzhi Zang</u>, William H. Wood, Dustin Schones, Kairong Cui, Tae-Young Roh, Brad Lhotsky, Robert P. Wersto, Weiqun Peng, Kevin G. Becker, Keji Zhao, Nan-ping Weng<br>
<i><b>Immunity</b></i> 30, 912&ndash;925 (2009)
</p></li><li><p>
<a href="http://www.cell.com/immunity/abstract/S1074-7613(08)00555-4" target="_blank">Global mapping of H3K4me3 and H3K27me3 reveals specificity and plasticity in lineage fate determination of differentiating CD4+ T cells</a><br>
Gang Wei, Lai Wei, Jinfang Zhu, <u>Chongzhi Zang</u>, Jane Hu-Li, Zhengju Yao, Kairong Cui, Yuka Kanno, Tae-Young Roh, Wendy Watford, Dustin E. Schones, Weiqun Peng, Hong-wei Sun, William E. Paul, John J. O’Shea, Keji Zhao<br>
<i><b>Immunity</b></i> 30, 155&ndash;167 (2009)
</p></li><li><p>
<a href="http://www.nature.com/ng/journal/v40/n7/full/ng.154.html" target="_blank">Combinatorial patterns of histone acetylations and methylations in the human genome</a><br>
Zhibin Wang*, <u>Chongzhi Zang</u>*, Jeffrey A. Rosenfeld*, Dustin E. Schones, Artem Barski, Suresh Cuddapah, Kairong Cui, Tae-Young Roh, Weiqun Peng, Michael Q. Zhang, Keji Zhao<br>
<i><b>Nature Genetics</b></i> 40, 897&ndash;903 (2008)
</p></li><li><p>
<a href="http://wulixb.iphy.ac.cn/EN/Y2006/V55/I1/299" target="_blank">Fluorescence measurement and acoustic diagnostics of plasma channels in air</a><br>
Zuo-Qiang Hao, Jie Zhang, Jin Yu, Zhe Zhang, Jia-Yong Zhong, <u>Chong-Zhi Zang</u>, Zhan Jin, Zhao-Hua Wang, Zhi-Yi Wei<br>
<i><b>Acta Physica Sinica</b></i> 55, 299&ndash;303 (2006)
</p></li></ol>

<hr>
<h3><font color="#002F6C">Software</font></h3>
<ul style="list-style-type:square; padding-left: 24px;"><li><p>
<b><a href="http://home.gwu.edu/~wpeng/Software.htm" target="_blank">SICER</a></b> (<u>S</u>patial-clustering <u>I</u>dentification of <u>C</u>hIP-<u>E</u>nriched <u>R</u>egions), a ChIP-Seq data analysis method. [<a href="https://academic.oup.com/bioinformatics/article/25/15/1952/212783" target="_blank">Publication</a>]
</p></li><li><p>
<b><a href="https://CRAN.R-project.org/package=MANCIE" target="_blank">MANCIE</a></b> (<u>M</u>atrix <u>A</u>nalysis and <u>N</u>ormalization by <u>C</u>oncordant <u>I</u>nformation <u>E</u>nhancement), a computational method for high-dimensional genomic data integration.  [<a href="http://www.nature.com/articles/ncomms11305" target="_blank">Publication</a>] 
</p></li><li><p>
<b><a href="http://cistrome.org/MARGE/" target="_blank">MARGE</a></b> (<u>M</u>odel-based <u>A</u>nalysis of <u>R</u>egulation of <u>G</u>ene <u>E</u>xpression), a comprehensive computational method for inference of cis-regulation of gene expression leveraging public H3K27ac genomic profiles in human or mouse.  [<a href="http://genome.cshlp.org/content/26/10/1417" target="_blank">Publication</a>]
</p></li><li><p>
<b><a href="http://faculty.virginia.edu/zanglab/bart/index.htm" target="_blank">BART</a></b> (<u>B</u>inding <u>A</u>nalysis for <u>R</u>egulation of <u>T</u>ranscription), a bioinformatics tool for predicting functional transcription factors (TFs) that bind at genomic cis-regulatory regions to regulate gene expression in the human or mouse genomes, given a query gene set or a ChIP-seq dataset as input.  [<a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty194/4956015" target="_blank">Publication</a>]
</p></li></ul>
<hr>
<h3><font color="#002F6C">Honors and Awards</font></h3>
<ul style="padding-left: 24px;">
<li><p>NIH/NCI Transition Career Development Award (2017&ndash;2020)
</p></li><li>
<p><a href="http://www.lls.org" target="_blank">Leukemia and Lymphoma Society</a> Fellow Award (2012&ndash;2015)
</p></li><li>
<p><a href="https://gwtoday.gwu.edu/significant-science" target="_blank">Dimitris N. Chorafas Foundation Prize</a> (2010)
</p></li><li>
<p><a href="http://www.sino-education.org/publish/portal131/tab6578/info125299.htm" target="_blank">Chinese Government Award for Outstanding Self-financed Student Abroad</a> (2008)
</p></li></ul>
<hr>
<h3><font color="#002F6C">Recruiting</font></h3>
<p><a href="http://faculty.virginia.edu/zanglab" target="_blank">My lab</a> is recruiting motivated young students and scholars to work on a variety of topics in computational biology in a collaborative research team. Postdocs, graduate students, and undergraduate students are all welcome. Please contact me for any questions.
</p><p>
Prospective postdocs can find the job details and submit applications <a href="http://jobs.virginia.edu/applicants/Central?quickFind=83436" target="_blank">here</a>.
</p>
<hr>
<p style="width: 500px; margin: auto;" align="center">
<br><br><i>"While the art of printing is left to us science can never be retrograde; what is once acquired of real knowledge can never be lost."<br><br>
&mdash;Thomas Jefferson, 1799</i><br><br>
</p>
<hr>
<p align="right">Last modified: February 5, 2019</p>
</div>

</body></html>