<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<link href="default.css" type="text/css" rel="stylesheet">
<link rel="stylesheet" href="ism/css/my-slider.css"/>
<script src="ism/js/ism-2.2.min.js"></script>
<script language="javascript" type="text/javascript">
function update_modtime() {
  var x = new Date(document.lastModified);
  document.getElementById("last_mod").innerHTML="Last modified: "+x;
}
</script>
<style>
body {margin:0;}
ul.topnav {
  list-style-type: none;
  margin: 0;
  padding: 0;
  overflow: hidden;
  background-color: #333;
}

ul.topnav li {float: left;}

ul.topnav li a {
  display: inline-block;
  color: #f2f2f2;
  text-align: center;
  padding: 4px 20px;
  text-decoration: none;
  transition: 0.3s;
  font-size: 17px;
}

ul.topnav li a:hover {background-color: #555;}

ul.topnav li.icon {display: none;}

@media screen and (max-width:680px) {
  ul.topnav li:not(:first-child) {display: none;}
  ul.topnav li.icon {
    float: right;
    display: inline-block;
  }
}

@media screen and (max-width:680px) {
  ul.topnav.responsive {position: relative;}
  ul.topnav.responsive li.icon {
    position: absolute;
    right: 0;
    top: 0;
  }
  ul.topnav.responsive li {
    float: none;
    display: inline;
  }
  ul.topnav.responsive li a {
    display: block;
    text-align: left;
  }
}
.column {
    float: left;
    width: 20%;
}
/* Clear floats after the columns */
.row:after {
    content: "";
    display: table;
    clear: both;
}
</style>
<title>Chongzhi Zang Lab</title>
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
</head>
<body bgcolor="white" onload="update_modtime();">
<div style="width: 900px; margin: auto;">
<img align="left" width="170" src="UVA-med-3.jpg">
<img align="right" width="150" src="cphg_logo.jpg">
<h2 style="font-size: 28pt" align="center"><a href="index.html" style="text-decoration:none;"><font color="#002F6C">Chongzhi Zang Lab&nbsp;&nbsp;</font></a></h2>

<ul class="topnav" id="myTopnav">
  <li><a href="index.htm">Home</a></li>
  <li><a class="active" href="people.htm">People</a></li>
  <li><a href="research.htm">Research</a></li>
  <li><a href="publication.htm">Publications</a></li>
  <li><a href="software.htm">Software</a></li>
  <li><a href="teaching.htm">Teaching</a></li>
  <li><a href="position.htm">Positions</a></li>
  <li><a href="contact.htm">Contact</a></li>
  <li class="icon">
    <a href="javascript:void(0);" style="font-size:15px;" onclick="myFunction()">☰</a>
  </li>
</ul>

<!--<script>
function myFunction() {
    var x = document.getElementById("myTopnav");
    if (x.className === "topnav") {
        x.className += " responsive";
    } else {
        x.className = "topnav";
    }
}
</script>
<div class="ism-slider" data-transition_type="fade" data-play_type="loop" id="home-slider">
  <ol>
    <li>
      <img src="ism/image/slides/_u/1488566891987_67648.jpg">
    </li>
    <li>
      <img src="ism/image/slides/_u/1488727852701_364381.jpg">
    </li>
    <li>
      <img src="ism/image/slides/_u/1488567032192_666768.jpg">
    </li>
    <li>
      <img src="ism/image/slides/_u/1488736436612_647904.png">
    </li>
  </ol>
</div>
-->
<h2 align="center"><font color="#002F6C">Research</font></h2>
<hr>
<h3><font color="#002F6C">News Releases</font></h3>
<ul style="list-style-type:disc; padding-left: 24px; line-height: 1.6;"><p>
<li><a href="https://news.virginia.edu/content/uva-develops-new-tools-battle-cancer-advance-genomics-research" target="_blank">UVA Develops New Tools to Battle Cancer, Advance Genomics Research</a>. <!--UVA Today. -->May 25, 2021</li>
<li><a href="https://news.virginia.edu/content/cancers-dangerous-renovations-our-chromosomes-revealed" target="_blank">Cancer’s Dangerous Renovations to Our Chromosomes Revealed</a>. <!--UVA Today. -->October 26, 2020</li>
</p></ul>
<hr>
<h3><font color="#002F6C">Research Description</font></h3>
<p style="line-height: 1.6;">
The research in the lab focuses on two aspects: 1) developing quantitative models and computational methods for analyzing high-throughput data generated from emerging genomics technologies; and 2) using innovative computational and data science approaches to study epigenetics and transcriptional regulation of gene expression in mammalian cell systems and human diseases such as cancer.
</p>
<p style="line-height: 1.6;">
How gene expression is regulated in chromatin is a fundamental question in molecular biology. The transcriptional program is a major determinant of cell identity; its dysregulation is involved in many diseases, including cancer. High-throughput genomics technologies enable us to obtain massive data measuring numerous factors and elements in the genome that affect chromatin states and gene regulation. We leverage big data and conduct computational research at the intersection of functional genomics, epigenetics, and cancer biology. Some research directions include:
</p>
<br>
<h3><font color="#002F6C">1. <i>Next-generation sequencing bioinformatics</i></font></h2>
<img align="right" src="images/sicer.jpg" style="width:240px">
<p style="width: 620px; line-height: 1.6;">
We are interested in developing innovative statistical methods and new algorithms for analyzing massive data from next-generation sequencing (NGS) coupled with various assays for studying genomic chromatin profiles, such as ChIP-seq for transcription factor and histone modification profiling, ATAC-seq and DNase-seq for chromatin accessibility profiling, etc. As a pioneer in ChIP-seq bioinformatics, we developed SICER (<a href="https://academic.oup.com/bioinformatics/article/25/15/1952/212783" target="_blank"><i>Bioinformatics</i> 2009</a>), one of the most widely used methods for ChIP-seq data analysis with exceptional performance for board histone marks. We are developing novel quantitative models for analyzing DNase/ATAC-seq data and for studying chromatin dynamics at both population and single-cell levels.
</p>
<br><br>
<h3><font color="#002F6C">2. <i>Chromatin, epigenetics, and transcriptional regulation</i></font></h2>
<img align="right" src="images/marge.jpg" style="width:240px">
<p style="width: 620px; line-height: 1.6;">
Our ultimate goal is to understand the fundamental mechanisms in transcriptional regulation and the functions of chromatin. We characterized dozens of histone modifications and histone modifying enzymes at the genomic scale (<a href="http://www.nature.com/ng/journal/v40/n7/full/ng.154.html" target="_blank"><i>Nat Genet</i> 2008</a>, <a href="http://www.nature.com/ng/journal/v41/n8/full/ng.409.html" target="_blank"><i>Nat Genet</i> 2009</a>, <a href="http://www.cell.com/abstract/S0092-8674(09)00841-1" target="_blank"><i>Cell</i> 2009</a>, <a href="http://www.cell.com/cell-stem-cell/abstract/S1934-5909(08)00582-1" target="_blank"><i>Cell Stem Cell</i> 2009</a>). Leveraging the large amount of publicly available ChIP-seq data, we developed several computational methods, including MARGE (<a href="http://genome.cshlp.org/content/26/10/1417" target="_blank"><i>Genome Res</i> 2016</a>) and BART (<a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty194/4956015" target="_blank"><i>Bioinformatics</i> 2018</a>), for predicting cis-regulatory elements and transcriptional regulators from differentially expressed gene sets using integrative learning approaches. We are specifically interested in studying functional enhancer regulation of gene expression in the high-order dynamic structure of chromatin in cancer and immune systems.
</p>
<br><br>
<h3><font color="#002F6C">3. <i>Genomic data integration for global regulatory networks</i></font></h2>
<img align="right" src="images/mancie.jpg" style="width:240px">
<p style="width: 620px; line-height: 1.6;">
High-dimensional genomic data analysis is challenging because of noises and biases in high-throughput experiments. We developed MANCIE (<a href="http://www.nature.com/articles/ncomms11305" target="_blank"><i>Nat Commun</i> 2016</a>), a method for bias correction and data integration of cross-platform genomic profiles on the same samples, using a Bayesian-supported principal component analysis (PCA)-based approach. We are interested in using statistical modeling and innovative data science approaches to integrate public multi-omics data for characterizing physical properties of mammalian chromatin structures and dynamic interactions between chromatin and DNA in human cell systems.
</p>
<br><br><br>
<hr>
<h3><font color="#002F6C">Collaborations</font></h3>
<p style="line-height: 1.6;">Computational biology is an interdisciplinary science. It cannot thrive without close collaborations between researchers with different backgrounds and expertise. We have established a collaborative research team and a cross-disciplinary environment in the lab, and we always commit to collaborations and team science on a variety of research projects with experimental biologists, clinicians, as well as statisticians, computer scientists, mathematicians, and physicists.</p>
<p style="line-height: 1.6;">Our collaborators include:
</p>
<ul style="list-style-type:disc; padding-left: 24px; line-height: 1.6;"><p>
<li><a href="https://med.virginia.edu/faculty/faculty-listing/gc8r/" target="_blank"><font color="#002F6C">Graham Casey</font></a>, Center for Public Health Genomics, University of Virginia</li>
<li><a href="https://med.nyu.edu/faculty/suresh-cuddapah" target="_blank"><font color="#002F6C">Suresh Cuddapah</font></a>, Department of Environmental Medicine, New York University</li>
<li><a href="http://www.stat.tsinghua.edu.cn/en/teambuilder/faculty/ke-deng/" target="_blank"><font color="#002F6C">Ke Deng</font></a>, Center for Statistical Science, Tsinghua University</li>
<li><a href="https://dutta-labwebsite.github.io/" target="_blank"><font color="#002F6C">Anindya Dutta</font></a>, Department of Genetics, University of Alabama at Birmingham</li>
<li><a href="https://med.virginia.edu/faculty/faculty-listing/fg5q/" target="_blank"><font color="#002F6C">Francine Garrett-Bakelman</font></a>, Department of Biochemistry and Molecular Genetics, University of Virginia</li>
<li><a href="http://guertinlab.org/" target="_blank"><font color="#002F6C">Michael J. Guertin</font></a>, Department of Genetics and Genome Sciences, University of Connecticut</li>
<li><a href="https://med.virginia.edu/faculty/faculty-listing/hj8d/" target="_blank"><font color="#002F6C">Hao Jiang</font></a>, Department of Biochemistry and Molecular Genetics, University of Virginia</li>
<li><a href="http://lilab.medicine.virginia.edu/" target="_blank"><font color="#002F6C">Hui Li</font></a>, Department of Pathology, University of Virginia</li>
<li><a href="https://uvahealth.com/findadoctor/profile/li-li" target="_blank"><font color="#002F6C">Li Li</font></a>, Department of Family Medicine, University of Virginia</li>
<li><a href="http://millerlab.org/" target="_blank"><font color="#002F6C">Clint L. Miller</font></a>, Center for Public Health Genomics, University of Virginia</li>
<li><a href="https://med.virginia.edu/faculty/faculty-listing/gm7sj/" target="_blank"><font color="#002F6C">Golam Mohi</font></a>, Department of Biochemistry and Molecular Genetics, University of Virginia</li>
<li><a href="http://www.ntziachristoslab.com/" target="_blank"><font color="#002F6C">Panagiotis Ntziachristos</font></a>, Department of Biochemistry and Molecular Genetics, Northwestern University</li>
<li><a href="https://med.virginia.edu/faculty/faculty-listing/ar7jq/" target="_blank"><font color="#002F6C">Aakrosh Ratan</font></a>, Center for Public Health Genomics, University of Virginia</li>
<li><a href="https://campuspress.yale.edu/wanglab/" target="_blank"><font color="#002F6C">Siyuan Steven Wang</font></a>, Department of Genetics, Yale University</li>
<li><a href="https://med.virginia.edu/yuh-hwa-wang-lab/" target="_blank"><font color="#002F6C">Yuh-Hwa Wang</font></a>, Department of Biochemistry and Molecular Genetics, University of Virginia</li>
<li><a href="https://hmh-cdi.org/our-team/xue-lab/" target="_blank"><font color="#002F6C">Hai-Hui Xue</font></a>, Center for Discovery and Innovation, Hackensack University Medical Center</li>
<li><a href="https://www.cvrc.virginia.edu/Yan/" target="_blank"><font color="#002F6C">Zhen Yan</font></a>, Center for Skeletal Muscle Research, Robert M. Berne Cardiovascular Research Center, University of Virginia</li>
<li><a href="https://engineering.virginia.edu/faculty/aidong-zhang" target="_blank"><font color="#002F6C">Aidong Zhang</font></a>, Department of Computer Science, University of Virginia</li>
<li><a href="https://www.stat.pitt.edu/people/tingting-zhang" target="_blank"><font color="#002F6C">Tingting Zhang</font></a>, Department of Statistics, University of Pittsburgh</li>
<li><a href="https://zhulab.wordpress.com/" target="_blank"><font color="#002F6C">Yuan Zhu</font></a>, Children's Research Institute, Children’s National Medical Center</li>
</p></ul>
<hr>
<h3><font color="#002F6C">Funding Support</font></h3>
<div class="row">
<div class="column">
<p>
<a href="https://www.nih.gov/" target="_blank"><img align="left" src="images/nih.png" style="width:110px"></a>
</p>
</div>
<div class="column">
<p>
<a href="http://www.chrb.org/" target="_blank"><img align="center" src="images/chrb.jpg" style="height:100px"></a>
</p>
</div>
<div class="column">
<p>
<a href="https://www.cancer.org/" target="_blank"><img align="center" src="images/acs.png" style="width:100px"></a>
</p>
</div>
<div class="column">
<p>
<a href="http://www.phibetapsi.org/national-project/" target="_blank"><img align="left" src="images/phibetapsi_logo.jpg" style="width:80px"></a>
</p>
</div>
<div class="column">
<p>
<br>
<a href="http://jktgfoundation.org/" target="_blank"><img align="left" src="images/jktg.png" style="width:110px"></a>
</p>
</div>
</div>
<hr>
<p align="right">Last modified: July 14, 2021</p>

</div>

</body></html>
