<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<link href="default.css" type="text/css" rel="stylesheet">
<link rel="stylesheet" href="ism/css/my-slider.css"/>
<script src="ism/js/ism-2.2.min.js"></script>
<script language="javascript" type="text/javascript">
function update_modtime() {
  var x = new Date(document.lastModified);
  document.getElementById("last_mod").innerHTML="Last modified: "+x;
}
</script>
<style>
body {margin:0;}
ul.topnav {
  list-style-type: none;
  margin: 0;
  padding: 0;
  overflow: hidden;
  background-color: #333;
}

ul.topnav li {float: left;}

ul.topnav li a {
  display: inline-block;
  color: #f2f2f2;
  text-align: center;
  padding: 4px 20px;
  text-decoration: none;
  transition: 0.3s;
  font-size: 17px;
}

ul.topnav li a:hover {background-color: #555;}

ul.topnav li.icon {display: none;}

@media screen and (max-width:680px) {
  ul.topnav li:not(:first-child) {display: none;}
  ul.topnav li.icon {
    float: right;
    display: inline-block;
  }
}

@media screen and (max-width:680px) {
  ul.topnav.responsive {position: relative;}
  ul.topnav.responsive li.icon {
    position: absolute;
    right: 0;
    top: 0;
  }
  ul.topnav.responsive li {
    float: none;
    display: inline;
  }
  ul.topnav.responsive li a {
    display: block;
    text-align: left;
  }
}
</style>
<title>Chongzhi Zang Lab</title>
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
</head>
<body bgcolor="white" onload="update_modtime();">
<div style="width: 900px; margin: auto;">
<img align="left" width="170" src="UVA-med-3.jpg">
<!--<img align="right" width="150" src="cphg_logo.jpg">-->
<h2 style="font-size: 28pt" align="center"><a href="index.html" style="text-decoration:none;"><font color="#002F6C">Chongzhi Zang Lab&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</font></a></h2>

<ul class="topnav" id="myTopnav">
  <li><a href="index.htm">Home</a></li>
  <li><a class="active" href="people.htm">People</a></li>
  <li><a href="research.htm">Research</a></li>
  <li><a href="publication.htm">Publications</a></li>
  <li><a href="software.htm">Software</a></li>
  <li><a href="teaching.htm">Teaching</a></li>
  <li><a href="position.htm">Positions</a></li>
  <li><a href="contact.htm">Contact</a></li>
  <li class="icon">
    <a href="javascript:void(0);" style="font-size:15px;" onclick="myFunction()">â˜°</a>
  </li>
</ul>

<h2 align="center"><font color="#002F6C">Software</font></h2>
<hr>
<table style="width:100%; line-height: 1.6;">
<tr>
<td style="width:120px; height:120px"><a href="bart/index.htm" target="_blank"><img src="images/bart_logo.jpg" style="width:100px"></a></td>
<td><p>
<b><a href="bart/index.htm" target="_blank">BART</a></b> (<u>B</u>inding <u>A</u>nalysis for <u>R</u>egulation of <u>T</u>ranscription), a bioinformatics tool for inferring functional transcription factors (TFs) that bind at genomic cis-regulatory regions to regulate gene expression in the human or mouse genomes, given a query gene set or a ChIP-seq dataset as input. [<a href="bart/index.htm" target="_blank">Package</a>] [<a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty194/4956015" target="_blank">Publication</a>]<br>
[<a href="http://bartweb.org" target="_blank"><b>BART web</b></a>]
</p></td>
</tr>
<tr>
<td style="width:120px; height:120px"><a href="https://github.com/zanglab/bart3d" target="_blank"><img src="images/bart3d_logo.jpg" style="width:100px"></a></td>
<td><p>
<b><a href="https://github.com/zanglab/bart3d" target="_blank">BART3D</a></b> (<u>BART</u> for <u>3D</u> genome data), a bioinformatics tool for inferring transcriptional regulators (TRs, including transcription factors and chromatin regulators) associated with differential chromatin interactions from Hi-C type 3D genome data for human or mouse. [<a href="https://github.com/zanglab/bart3d" target="_blank">Package</a>] [<a href="https://doi.org/10.1093/bioinformatics/btab173" target="_blank">Publication</a>]
</p></td>
</tr>
<tr>
<td style="width:120px; height:120px"><a href="http://cistrome.org/MARGE/" target="_blank"><img src="images/marge_logo.jpg" style="width:100px"></a></td>
<td><p>
<b><a href="http://cistrome.org/MARGE/" target="_blank">MARGE</a></b> (<u>M</u>odel-based <u>A</u>nalysis of <u>R</u>egulation of <u>G</u>ene <u>E</u>xpression), a comprehensive computational method for inference of cis-regulation of gene expression leveraging public H3K27ac genomic profiles in human or mouse. [<a href="http://cistrome.org/MARGE/" target="_blank">Package</a>] [<a href="http://genome.cshlp.org/content/26/10/1417" target="_blank">Publication</a>]
</p></td>
</tr>
<tr>
<td style="width:120px; height:120px"><a href="https://CRAN.R-project.org/package=MANCIE" target="_blank"><img src="images/mancie_logo.jpg" style="width:100px"></a></td>
<td><p>
<b><a href="https://CRAN.R-project.org/package=MANCIE" target="_blank">MANCIE</a></b> (<u>M</u>atrix <u>A</u>nalysis and <u>N</u>ormalization by <u>C</u>oncordant <u>I</u>nformation <u>E</u>nhancement), a computational method for high-dimensional genomic data integration. [<a href="https://CRAN.R-project.org/package=MANCIE" target="_blank">Package</a>] [<a href="http://www.nature.com/articles/ncomms11305" target="_blank">Publication</a>] 
</p></td>
</tr>
<tr>
<td style="width:120px; height:120px"><a href="https://zanglab.github.io/SICER2" target="_blank"><img src="images/recognicer_logo.jpg" style="width:100px"></a></td>
<td><p>
<b><a href="https://zanglab.github.io/SICER2" target="_blank">RECOGNICER</a></b> (<u>Re</u>cursive <u>co</u>arse-<u>g</u>rai<u>n</u>ing <u>i</u>dentification for <u>C</u>hIP-seq <u>e</u>nriched <u>r</u>egions), a ChIP-seq ultra-broad peak calling method. [<a href="https://link.springer.com/article/10.1007%2Fs40484-020-0225-2" target="_blank">Publication</a>] [<a href="https://github.com/zanglab/recognicer" target="_blank">Source code</a>] [Package built in <a href="https://zanglab.github.io/SICER2" target="_blank">SICER2</a>] 
</p></td>
</tr>
<tr>
<td style="width:120px; height:120px"><a href="https://github.com/zang-lab/SELMA" target="_blank"><img src="images/selma_logo.jpg" style="width:100px"></a></td>
<td><p>
<b><a href="https://github.com/zang-lab/SELMA" target="_blank">SELMA</a></b> (<u>S</u>implex <u>E</u>ncoded <u>L</u>inear <u>M</u>odel for <u>A</u>ccessible chromatin), a computational model for estimation and correction of intrinsic enzymatic cleavage biases in bulk and single-cell chromatin accessibility profiling (DNase-seq and ATAC-seq) data. [<a href="https://github.com/zang-lab/SELMA" target="_blank">Package</a>] [<a href="https://www.nature.com/articles/s41467-022-33194-z" target="_blank">Publication</a>]
</p></td>
</tr>
<tr>
<td style="width:120px; height:120px"><a href="https://zanglab.github.io/SICER2" target="_blank"><img src="images/sicer_logo.jpg" style="width:100px"></a></td>
<td><p>
<b><a href="https://zanglab.github.io/SICER2" target="_blank">SICER</a></b> (<u>S</u>patial-clustering <u>I</u>dentification of <u>C</u>hIP-<u>E</u>nriched <u>R</u>egions), a ChIP-Seq data analysis method. [<a href="http://home.gwu.edu/~wpeng/Software.htm" target="_blank">Package (v1.1)</a>] [<a href="https://academic.oup.com/bioinformatics/article/25/15/1952/212783" target="_blank">Publication</a>]<br><br>
<a href="https://zanglab.github.io/SICER2" target="_blank"><b>SICER2</b></a> [<a href="https://zanglab.github.io/SICER2" target="_blank">Package (v2.0)</a>]
</p></td>
</tr>
</table>

<hr>
<h2 align="center"><font color="#002F6C">Resources</font></h2>
<hr>
<table style="width:100%; line-height: 1.6;">
<tr>
<td style="width:120px; height:120px"><a href="bartcancer" target="_blank"><img src="images/bartcancer_logo.jpg" style="width:100px"></a></td>
<td><p>
<b><a href="bartcancer" target="_blank">BART Cancer</a></b>, a database resource for computationally predicted transcriptional regulator activities in 15 human cancers from The Cancer Genome Atlas (TCGA). [<a href="bartcancer" target="_blank">Web resource</a>] [<a href="https://academic.oup.com/narcancer/article/3/1/zcab011/6180061" target="_blank">Publication</a>]
</p></td>
</tr>
</table>
<hr>
<p align="right">Last modified: July 24, 2024</p>
</div>

</body></html>
